docker/conf/docker-compose-win.yml.dist

```diff
@@ -96,6 +96,14 @@ services:
         environment:
             - discovery.type=single-node
 
+    kibana:
+        image: docker.elastic.co/kibana/kibana-oss:7.5.1
```

Comment:
I found that current version is 7.6 (https://www.elastic.co/guide/en/kibana/current/index.html). Don't we change it yet?

Reply:
It didn't exists 1 month ago, when I created this PR... If you want, upgrade this version...

Reply:
I know it :) It was question for @s3tezsky :)